pkgs/by-name/su/supermariowar/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, enet
+, yaml-cpp
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "supermariowar";
+  version = "2.0";
```

Comment:
Since we are using a commit hash instead of a tag in `rev`, we should change the version name to fit appropriately. `2.0-unstable-YYYY-MM-DD` should work fine here, the date is the date of the commit you are using.

Reply:
Ok, it should be better now, I changed from version `2.0` to `2.0-unstable-2020-10-13`.